tests/pytests/functional/modules/test_win_pkg.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+    pytest.mark.skip_unless_on_windows,
+    pytest.mark.slow_test,
+]
+
+
+@pytest.fixture(scope="module")
+def pkg_def_contents(state_tree):
```

Comment:
```suggestion
def pkg_def_contents():
```
Not using the fixture

Reply:
Can be fixed at another time, it's not a real problem.